src/main/java/Duke.java

```diff
@@ -1,10 +1,184 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    private static Task[] taskList = new Task[100];
+    private static String[] storeUserText = new String[100];
+    private static boolean doExit = false;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        lineSeparator();
+        greet();
+        lineSeparator();
+
+        while (!doExit) {
+            String userInput = getInput();
+            try {
+
+                String[] userInputArray = checkInput(userInput);
+
+            //firstWord will decide the actions that will be done
+            String firstWord = userInputArray[0];
+
+            if (firstWord.equals("bye")) {
+                doExit = true;
+                lineSeparator();
+                break;
+            }
+
+            if (firstWord.equals("list")) {
+                list();
+            }
+            else if (firstWord.equals("done")) {
+                markDone(userInputArray);
+            }
+            else if (firstWord.equals("todo")) {
+                markToDo(userInput);
+            }
+            else if (firstWord.equals("deadline")) {
+                markDeadline(userInput);
+            }
+            else if (firstWord.equals("event")) {
+                markEvent(userInput);
+            }
+            else {
+                addTask(storeUserText, userInput);
+            }
+
+            } catch (DukeException e){
+                errorMessage(userInput);
+            }
+
+            }
+
+        }
+
+    private static String getInput() {
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine();
+        return userInput;
+    }
+
+    private static String[] checkInput(String userInput) throws DukeException{
+        String[] checkUserInput = userInput.split(" ");
+        if(checkUserInput.length < 2) {
+            throw new DukeException();
+        }
+        return checkUserInput;
+    }
+
+    private static void errorMessage(String userInput) {
+        lineSeparator();
+        switch(userInput) {
+            case "todo":
+            case "deadline":
+            case "event":
+                System.out.println("☹ OOPS!!! The description of a " + userInput + " cannot be empty.");
+                break;
+            case "bye":
+                doExit = true;
+                goodbye();
+                break;
+            default:
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+        }
+        lineSeparator();
+    }
```

Comment:
The number of cases with the same outcome doesn't look that nice. Maybe you can consider implementing a boolean variable that checks if it is either one of "todo", "deadline" or "event" and print if the boolean flag detects true. This allows you to name the boolean variable appropriately and your code will be more comprehensive.